doc/examples/centralized_logging/README.md

```diff
@@ -0,0 +1,89 @@
+# ELK and Harmony AP Example Environment
+
+This example demonstrates how to deploy a secure ELK stack alongside a Harmony AP application, using Docker Compose for orchestration.
+
+## Overview
+
+This environment shows how to:
+
+1. Generate a self-signed CA and TLS certificates for Elasticsearch.
+2. Initialize Elasticsearch with built-in users, roles, and TLS encryption.
+3. Run Elasticsearch as a single-node cluster.
+4. Deploy Kibana connected to the secured Elasticsearch instance.
+5. Configure Logstash for JSON and GELF input, with structured filtering for application logs.
+6. Run a MySQL database for Harmony AP data storage.
+7. Launch the Harmony AP service with environment-driven configuration and optional GELF logging.
+
+All services are defined in a single docker-compose.yml file and coordinated using Docker Compose.
+
+## Prerequisites
```

Comment:
Are there any requirements for the host OS? For example, should the example work on Windows, Linux and MacOS?

Reply:
It's also OK to limit the supported platforms (e.g., only Linux and MacOS are supported), but we should just mention in the prerequisites.